src/file_analysis/Manager.cc

```diff
@@ -417,14 +406,15 @@ bool Manager::RemoveFile(const string& file_id)
 
 	f->EndOfFile();
 	delete f;
-	id_map.Remove(&key);
-	delete static_cast<bool*>(ignored.Remove(&key));
+	id_map.erase(file_id);
+
+	ignored.erase(file_id);
 	return true;
 	}
 
 bool Manager::IsIgnored(const string& file_id)
 	{
-	return ignored.Lookup(file_id.c_str()) != 0;
+	return ignored.count(file_id) != 0;
```

Comment:
Is there an advantage to using `count` here instead of `return ignored.find(file_id) != ignored.end()` [StackOverflow](https://stackoverflow.com/questions/25490357/checking-for-existence-in-stdmap-count-vs-find) seems to think this might be a little cleaner.

Reply:
Ah yes, `count` may iterate over the whole `set`. Good catch. C++20 adds `contains` finally.